test/test_runtime.py

```diff
@@ -26,6 +26,12 @@ def test_runtime_version(runtime: Runtime) -> None:
     assert version == runtime.version
 
 
+def test_runtime_version_outdated() -> None:
```

Comment:
I think this test is a good candidate for `@pytest.mark.parametrize` — this way you'll document which types of values you had in mind when adding this feature.

Reply:
I go for parametrize when there is a clear benefit in avoiding duplication, but for cases like this I prefer not using it. The reason behind is that parameterized tests are harder to read and to run as you need to add extra quotes on shell due to the use of `[...]` in their ids. Anyway, we can easily rewrite them when needed.

Reply:
No need to merge this with other tests. Just add something other than 99, like corner cases.